scss/components/_cards.scss

```diff
@@ -0,0 +1,107 @@
+.home-section {
```

Comment:
How home section is related to cards? Suggest to create separate scss for home design.